packages/create-pantheon-decoupled-kit/src/templates/next-wp-search-api-addon/pages/search/[[...searchAlias]].jsx

```diff
@@ -1,26 +1,95 @@
 import { NextSeo } from 'next-seo';
+import Link from 'next/link';
 import Layout from '../../components/layout';
 import PageHeader from '../../components/page-header';
 import { getFooterMenu } from '../../lib/Menus';
+import { getSearchedPosts } from '../../lib/Posts.js';
+import styles from './searchPage.module.css';
 
-export default function PageTemplate({ menuItems }) {
+export default function PageTemplate({
+	menuItems,
+	searchResults,
+	error,
+	expectedResults,
+}) {
 	return (
 		<Layout footerMenu={menuItems}>
 			<NextSeo
 				title="Decoupled Next WordPress - Search"
 				description="Generated by create-pantheon-decoupled-kit."
 			/>
 			<PageHeader title="Search Results" />
+			{error ? (
+				<div className={styles.altResult}>
+					<span>An error occurred while fetching search results</span>
+				</div>
+			) : (
+				<section className={styles.section}>
+					<div className={styles.container}>
+						{searchResults?.length > 0 ? (
+							<ul>
+								{searchResults?.map(({ title, excerpt, uri, postId }) => (
```

Comment:
Forgot to mention this on the review: There could be a bit more space between the results, looks a bit crowded right now.
![Screen Shot 2023-08-22 at 9 13 28 AM](https://github.com/pantheon-systems/decoupled-kit-js/assets/63015754/5b0abb13-10ff-471f-831d-0a8875d03d3d)